transaction/src/manifest/ast.rs

```diff
@@ -227,22 +227,10 @@ pub enum Instruction {
         address: Value,
         args: Vec<Value>,
     },
-    SetAndLockOwnerRole {
```

Comment:
I see we've removed SetAndLockOwnerRole but are keeping LockOwnerRole - I think this is fine tbh, you can do a Set and then the new Owner can choose to lock if they wish. Perhaps that's better even.

Reply:
Yup, exactly